lottie-ios.podspec

```diff
@@ -27,7 +27,7 @@ Lottie enables designers to create and ship beautiful animations without an engi
   s.osx.deployment_target = '10.12'
   s.tvos.deployment_target = '11.0'
 
-  s.source_files = 'Sources/**/*'
+  s.source_files = 'Sources/**/*.swift'
   s.exclude_files = 'Sources/**/*.md'
```

Comment:
This is unnecessary now that we only include Swift files, right?

```suggestion
```

Reply:
Yep, that's right.